management/client.go

```diff
@@ -91,6 +91,37 @@ type ClientJWTConfiguration struct {
 	Algorithm string `json:"alg,omitempty"`
 }
 
+type Addons struct {
+	Aws                  map[string]string `json:"aws,omitempty"`
+	AzureBlob            map[string]string `json:"azure_blob,omitempty"`
+	AzureSb              map[string]string `json:"azure_sb,omitempty"`
+	Rms                  map[string]string `json:"rms,omitempty"`
```

Comment:
Should we prefer `map[string]interface{}` which leaves us more room in case an addon needs more complex data structure?